modules/core/module.go

```diff
@@ -197,7 +197,10 @@ func (AppModule) ConsensusVersion() uint64 { return 7 }
 
 // BeginBlock returns the begin blocker for the ibc module.
 func (am AppModule) BeginBlock(ctx context.Context) error {
-	ibcclient.BeginBlocker(sdk.UnwrapSDKContext(ctx), am.keeper.ClientKeeper)
+	if err := ibcclient.BeginBlocker(sdk.UnwrapSDKContext(ctx), am.keeper.ClientKeeper); err != nil {
+		// Log the error but do not return it to avoid halting the chain
+		fmt.Printf("non-critical error in IBC BeginBlocker: %v\n", err)
```

Comment:
fmt is not the correct way to log errors in our modules. Take a look at how it is done elsewhere to find some examples :)